pkg/exchange/coinbase/exchage.go

```diff
@@ -145,18 +145,20 @@ func (e *Exchange) SubmitOrder(ctx context.Context, order types.SubmitOrder) (cr
 		req.Price(order.Price)
 	}
 
-	// set time in force
-	switch order.TimeInForce {
-	case types.TimeInForceGTC:
-		req.TimeInForce("GTC")
-	case types.TimeInForceIOC:
-		req.TimeInForce("IOC")
-	case types.TimeInForceFOK:
-		req.TimeInForce("FOK")
-	case types.TimeInForceGTT:
-		req.TimeInForce("GTT")
-	default:
-		return nil, fmt.Errorf("unsupported time in force: %v", order.TimeInForce)
+	// set time in force, using default if not set
+	if len(order.TimeInForce) > 0 {
```

Comment:
Could you set a default timeInForce beforehand?

Reply:
I want to leave it unset if it's not given by to user.
So it will fallback to Coinbase API defaults.

I want it to be consistent with Coinbase API so I didn't set a default value here.